app/full_text_search.py

```diff
@@ -130,6 +152,12 @@ def create_jp_name_fields(**kwargs):
                         name=field+'_romanized_by_jp_name_dict',
                         value=romanized_japanese_name)
                 )
+                connected_romanized_jp_name = ''
+                for x in range(5):
```

Comment:
This can be simplified by:

    ''.join(romanized_japanese_name for _ in range(5))

_ is a commonly used name of a variable which is unused.

Also, wouldn't this results in a string like 'yamadayamadayamadayamada'? Don't you need a whitespace?

Also, please define a constant for 5 and use it here instead of directly using 5.